app/src/components/common/__tests__/Toast.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
+import Toast from '@/components/common/Toast.vue';
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import {
+  faCheckCircle,
+  faInfoCircle,
+  faExclamationTriangle,
+  faCircleXmark
+} from '@fortawesome/free-solid-svg-icons'
+
+library.add(faCheckCircle, faCircleXmark, faInfoCircle, faExclamationTriangle)
+const localVue = createLocalVue()
+localVue.component('FontAwesomeIcon', FontAwesomeIcon)
+
+describe('Toast.vue', () => {
+  let wrapper: Wrapper<Vue>
+
+  beforeEach(() => {
+    wrapper = mount(Toast, {
+      slots: {
+        default: 'Hello, I\'m toast!',
+      },
+      localVue
+    });
+  });
+
+  afterEach(() => {
+    wrapper.destroy();
+  });
+
+  it('renders the component with default "success" style', async () => {
+    await wrapper.setData({ show: true });
+    expect(wrapper.classes('success')).toBe(true);
+  });
+
+  it('renders the component with custom style', async () => {
+    await wrapper.setProps({ styleType: 'warning' });
+    await wrapper.setData({ show: true });
+    expect(wrapper.classes('warning')).toBe(true);
+  });
+
+  it('calculates animation duration in seconds based on timeout', async () => {
+    await wrapper.setData({ timeout: 1000 });
+    expect((wrapper.vm as any).getAnimationDuration).toBe('1s');
```

Comment:
it can be shortened by declaring a variable and assigning it wrapper.vm

Something like:  
let vm: Toast | any
...
vm = wrapper.vm as Toast

Reply:
Added the variable in recent commit